addons/account/models/account.py

```diff
@@ -177,6 +193,28 @@ def action_open_reconcile(self):
         }
 
 
+class AccountGroup(models.Model):
+    _name = "account.group"
+
+    _parent_store = True
+
+    parent_id = fields.Many2one('account.group', index=True, ondelete='cascade')
+    parent_left = fields.Integer('Left Parent', index=True)
+    parent_right = fields.Integer('Right Parent', index=True)
+    name = fields.Char(required=True)
+    code_prefix = fields.Char()
+
+    @api.depends('name', 'code_prefix')
```

Comment:
I copied a `name_get` from somewhere else and it had the depends, so I just updated them thinking it did something. I guess it doesn't, so I removed it. There's multiple cases (~8) where these are set: https://github.com/odoo/odoo/blob/d11df4f7b76cfc6bf5fdaaf0eaa20b107a583c4d/addons/account/models/account.py#L159, https://github.com/odoo/odoo/blob/d11df4f7b76cfc6bf5fdaaf0eaa20b107a583c4d/addons/account/models/account.py#L502, https://github.com/odoo/odoo/blob/cd1a0fcba9fc32b89aaa3f93a8df5ede00ae84d9/addons/event/models/event.py#L200, ...

Can you think of a reason why?